components/kyma-environment-broker/internal/process/provisioning/lms_cert_step.go

```diff
@@ -173,7 +173,7 @@ func (s *lmsCertStep) Run(operation internal.ProvisioningOperation, logger logru
 [FILTER]
         Name record_modifier
         Match *
-        Record cluster_name %s
+        Record subaccount_id %s
```

Comment:
pr which is also changing that: https://github.com/kyma-incubator/compass/pull/1218/files so maybe it should be aligned between them

Reply:
I've added a comment there yesterday.